app/src/main/java/co/nimblehq/compose/ecommerce/ui/bottomnavigationbar/NavigationItem.kt

```diff
@@ -0,0 +1,39 @@
+package co.nimblehq.compose.ecommerce.ui.bottomnavigationbar
+
+import androidx.annotation.DrawableRes
+import co.nimblehq.compose.ecommerce.R
+
+sealed class NavigationItem(
+    val route: String,
+    @DrawableRes val iconResId: Int,
+    @DrawableRes val selectedIconResId: Int,
+    val title: String
+) {
+    object Home : NavigationItem(
```

Comment:
Any reason why we should use `object` class instead of `data` class, Seems like we can't update or apply the value inside  the `object` class 🤔

Reply:
@Wadeewee It's redundant to use `data class` here, we have to define the properties on `data class` then pass the properties to the `sealed class` for the abstract type. And all the items here is not changed in the run time 😄